packages/components/package.json

```diff
@@ -32,6 +32,7 @@
     "@use-it/event-listener": "^0.1.3",
     "classnames": "^2.2.6",
     "color": "^3.1.2",
+    "filesize": "^6.1.0",
```

Comment:
This is used to give us human readable sizes based on the file size provided in bytes.